contracts/ERC721M.sol

```diff
@@ -443,11 +448,13 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
         revert InvalidStage();
     }
 
+    function getTimestampExpirySeconds() public view override returns (uint64) {
+        return _timestampExpirySeconds;
+    }
+
     function _assertValidTimestamp(uint64 timestamp) internal view {
-        uint64 threshold = msg.sender == _crossmintAddress
-            ? CROSSMINT_TIMESTAMP_EXPIRY_SECONDS
-            : MIN_STAGE_INTERVAL_SECONDS;
-        if (timestamp < block.timestamp - threshold) revert TimestampExpired();
+        if (timestamp < block.timestamp - getTimestampExpirySeconds())
```

Comment:
Do you think we should have a different expiry for crossmint?

e.g. add 2 mins on top of getTimestampExpirySeconds()

Reply:
I think it will be fine, as long as we pick a large enough expiry. That way we save some gas since we don't have to check `msg.sender` or store a separate expiry variable for crossmint